shuup/admin/static_src/base/js/main-menu.js

```diff
@@ -58,14 +58,42 @@ const handleMainMenu = () => {
       mainMenu.classList.remove('open');
     });
   }
+  var desktop = false;
+  function closeAdminMenu() {
+    $('#top-header').css({ 'left' : '0px', 'width' : '100%'});
+    $('.support-nav-wrap').css({ 'padding-left': '30px' });
+    $('#main-content').css({ 'margin-left': '0px' });
+  }
+  function openAdminMenu() {
+    $('#top-header').css({ left: '280px', width : 'calc(100% - 280px)' });
+    $('.support-nav-wrap').css({ 'padding-left': '310px' });
+    $('#main-content').css({ 'margin-left': '280px' });
+  }
+  // media query change
+  function widthChange() {
+    if(($(window).width() > 768) && (mainMenu.classList.contains('open'))) {
+      openAdminMenu();
+    } else {
+      closeAdminMenu();
+    }
+  }
+
+  window.addEventListener("resize", widthChange);
+  if ($(window).width() >= 1024) {
+    $('.main-menu').addClass('open');
+  }
```

Comment:
RE: all above:
`body` can take `class` definitions as well. Which means, you can add (for example) `menu-open` class to body. So, your togglemenu could only do `$("body").toggleClass("menu-open")` instead of all this. Maybe?

Then, in styles define `.menu-open #top-header { left: xxx, width: yyy }` etc.